### ISSUE TITLE
start to command

### ISSUE THREAD

**Issue by AnimeBubsi** [propstg/ambulance-mission#28]
i want it start with command not control pressed can someone help me :)

**Comment by propstg** [comment]
Hey, sorry for the delay.  I don't have FiveM installed, so I can't test it, but I think the linked pull request might get you close? I don't plan on merging that change, but you can check the branch out and try it.

**Comment by AnimeBubsi** [comment]
okay thanks for answering but its just easier and cooler with command bec you cant miss click it :) but have nice day or night i dont know :)

**Comment by propstg** [comment]
If the branch works, I could maybe add it as a config option. Let me know how #29 works, if you try it. Have a nice day/night, too.

**Comment by AnimeBubsi** [comment]
yeah it works thank you so much its much better than that button :)

**Comment by AnimeBubsi** [comment]
then you have to delete this line  " elseif not playerData.isInAmbulance and newPlayerData.isInAmbulance then
        ESX.ShowHelpNotification(Wrapper._('start_game_help'))
" so that doesent show you that notification "press something to start npc job"

**Comment by propstg** [comment]
Cool, glad it works. And good catch with the message. I'll try to get around to making it configurable soonish, after I reinstall FiveM.

**Comment by AnimeBubsi** [comment]
okay hopefully you get it work and sorry for bad english hopefully you understanded something :D

**Comment by propstg** [comment]
Alright, well, I installed FiveM, but I can't get my old server to start. I applied the `help wanted` label so this doesn't get auto-closed.
